src/main/java/duke/command/HelpCommand.java

```diff
@@ -0,0 +1,27 @@
+package duke.command;
+
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+/**
+ * HelpCommand prints help list.
+ *
+ * @author Chng Zi Hao
+ */
+public class HelpCommand extends Command {
```

Comment:
Maybe can be renamed to CommandHelper which is more suitable to be a Class name?